src/quo/components/list_items/saved_address/view.cljs

```diff
@@ -32,7 +33,7 @@
         {:size   :paragraph-2
          :weight :monospace
          :style  (style/account-address blur? theme)}
-        address]]]]))
+        (or ens (address/get-shortened-key address))]]]]))
```

Comment:
curious if we should handle this on status-im side too? 🤔

Reply:
I am not sure, if we'll always receive the full address maybe it makes sense to shorten in the quo component

Reply:
me neither 🤷‍♂️ , we can always revisit later if needs be

Reply:
I guess we'll find the best approach once we start to render components with real data, for now it serves its purpose I think